<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RestyCageWindow</class>
 <widget class="QMainWindow" name="RestyCageWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RESTy Cage</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/icons/corona-red.ico</normaloff>:/icons/corona-red.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QToolBox" name="toolBox">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="collectionsPage">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>868</width>
             <height>520</height>
            </rect>
           </property>
           <attribute name="icon">
            <iconset theme="folder"/>
           </attribute>
           <attribute name="label">
            <string>Collections</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="newCollectionBtn">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset theme="list-add"/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="removeCollectionBtn">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset theme="list-remove"/>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="newQueryBtn">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset theme="list-add"/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTreeView" name="collectionsTreeView">
              <property name="editTriggers">
               <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
              </property>
              <attribute name="headerVisible">
               <bool>false</bool>
              </attribute>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="environmentsPage">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>868</width>
             <height>520</height>
            </rect>
           </property>
           <attribute name="icon">
            <iconset theme="document-page-setup"/>
           </attribute>
           <attribute name="label">
            <string>Environments</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="addEnvironmentBtn">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset theme="list-add"/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="delEnvBtn">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset theme="list-remove"/>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTreeView" name="envsTreeView">
              <property name="editTriggers">
               <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
              </property>
              <attribute name="headerVisible">
               <bool>false</bool>
              </attribute>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
